src/interfaces/javaapi/javaapi.go

```diff
@@ -118,12 +118,13 @@ const (
 
 	edgeDir = "/data/user/0/com.samsung.orchestration.service/files/"
```

Comment:
Remove "/" at the last .

**As-is:**
"/data/user/0/com.samsung.orchestration.service/files/"

**To-be:**
"/data/user/0/com.samsung.orchestration.service/files"